### PR TITLE
return last-modified in resource properties

### PR DESCRIPTION
**Pull request by malaffoon**
As part of the phase 6 requirement 5 changes, i want to use the archive's last-modified to keep track of when a custom template was updated. I want to do this to allow for manual manipulation of the files. Plus it avoids separate metadata storage.

### CODE REVIEW

File: archive/src/test/java/org/opentestsystem/rdw/archive/LocalArchiveServiceTest.java [hunk 1]
```diff
@@ -23,7 +24,7 @@
     @Rule
     public TemporaryFolder temporaryFolder = new TemporaryFolder();
     private ArchiveService service;
-    private String testPrefix = "test-prefix";
+    private final String testPrefix = "test-prefix";
```
Comment (malaffoon) [inline archive/src/test/java/org/opentestsystem/rdw/archive/LocalArchiveServiceTest.java:27]: fix warning

File: archive/src/test/java/org/opentestsystem/rdw/archive/LocalArchiveServiceTest.java [hunk 2]
```diff
@@ -33,13 +34,6 @@ public void createService() {
         service = new ArchiveServiceDecorator(archivePropertiesTenant);
     }
 
-    private String addPrefix(String location) {
-        if (testPrefix == null || testPrefix.equals("")) {
-            return location;
-        }
-        return "/" + testPrefix + location;
-    }
-
```
Comment (malaffoon) [inline archive/src/test/java/org/opentestsystem/rdw/archive/LocalArchiveServiceTest.java:42]: fix warning

**Comment by malaffoon** [comment]
I'm going to commit this so i can move forward with my RDW_Reporting work. I'll address any comments.

File: archive/src/test/java/org/opentestsystem/rdw/archive/LocalArchiveServiceTest.java (hunk 2 above)
Comment (gregcharles) [inline archive/src/test/java/org/opentestsystem/rdw/archive/LocalArchiveServiceTest.java:42]: Hmm, I wonder why this was here. To make it easier to test with no testPrefix? Weird. Well, good riddance to bad rubbish.

**Pull request merged**